app/queries/proxy_rule_query.rb

```diff
@@ -1,7 +1,8 @@
 # frozen_string_literal: true
 
 class ProxyRuleQuery
-  def initialize(owner_type:, owner_id:, sort: nil, direction: nil, per_page: ThreeScale::Search::Helpers::MAX_PER_PAGE)
+  def initialize(owner_type:, owner_id:, sort: nil, direction: nil,
+                 per_page: ThreeScale::Search::Helpers::MAX_SEARCH_PAGE_SIZE)
```

Comment:
Purely cosmetic but IMO arguments should go in the same line, they shouldn't have more than 1-2 anyway. On the other hand adding a new line makes the actual change harder to see.

Reply:
Don't we care about the line length?
I see that the the [rule](https://github.com/3scale/porta/blob/e93ebed/.rubocop.yml#L59-L61) is disabled, but nevertheless, don't we want to keep them under control?

I normally wrap the lines > 120 chars. WDYT?

Reply:
+1 for a max line length.
> I normally wrap the lines > 120 chars

That's what I usually do as well. Max of 120 chars.

Reply:
Resolved by removing that argument from the initializer.

Reply:
I'm also up for max line length. I hate multi-line format though, I prefer either 1 line or 1-per-line.